guide/payments/coin-selection.md

```diff
@@ -0,0 +1,312 @@
+
+### Optimisations
+
+Coin selection will always lead to certain effects and outcomes which can either optimise or hinder our bitcoin payments. Below is a breakdown of the three most common effects of coin control.
+
+#### Cost
+
+Miners select transactions on the basis of their fee, measured by the fee rate of satoshis per byte. Therefore, it is cost effective for users to minimize transaction size by minimising the number of inputs.
+
+#### Speed
+
+Higher fees will increase the likelihood of your transaction being prioritised by miners and included in the next block, thereby increasing the speed of your transaction. This can be optimised by either increasing the fee rate, or by selecting a higher number of transaction inputs.
+
+#### Privacy
+
+Because each UTXO can be traced backwards on a public ledger, we can unearth the digital footprints of bitcoin payments. Privacy is often lost due to the exposure of unknown coins, addresses, balances, and economic activity to unrecognised trading partners. Privacy can be optimised by selecting transaction inputs from an anonymity set (CoinJoins or mixers) or a recognised label or cluster, or by reducing the size of change outputs. 
+
+## Automatic coin selection
+
+This strategy has arguably become the most popular form of coin selection used in bitcoin wallets and transactions. Wallet applications are delegated control, selecting coins on behalf of users in order to fulfil a payment request. As mentioned above, “fractions (of bitcoins) are retrieved by an algorithm to fulfill the payment amount”. This algorithm varies from wallet to wallet, and is dependent upon the desired outcome of optimisation.
+
+Automatic selection strategies can provide a near frictionless user experience, and is therefore suitable for most introductory level bitcoin wallets.
+
+{% include picture.html
+   image = "/assets/images/guide/glossary/coin-selection/automatic-coin-selection.jpg"
+   retina = "/assets/images/guide/glossary/coin-selection/automatic-coin-selection@2x.jpg"
+   mobile = "/assets/images/guide/glossary/coin-selection/automatic-coin-selection-mobile.jpg"
+   mobileRetina = "/assets/images/guide/glossary/coin-selection/automatic-coin-selection-mobile@2x.jpg"
+   alt-text = "Automatic Coin Selection"
+   width = 800
+   height = 400
+%}
+
+### How it works
+A user chooses to send a payment to one of their contacts. They enter the amount of bitcoin they wish to send, select their transaction fee rate, and approve the outgoing payment request. During this process, their wallet selects which coins to use to fund the transaction’s inputs, making sure it can fulfill the payment request.
```

Comment:
```suggestion
A user chooses to send a payment to one of their contacts. They enter the amount of bitcoin they wish to send, select their transaction fee rate, and approve the outgoing payment request. During this process, their wallet **automatically** selects which coins to use to fund the transaction’s inputs, making sure it can fulfill the payment request.
```